adal/src/main/java/com/microsoft/aad/adal/Logger.java

```diff
@@ -40,69 +38,125 @@
  */
 public class Logger {
     private static Logger sINSTANCE = new Logger();
-    static final String DATE_FORMAT = "yyyy-MM-dd HH:mm:ss";
 
-    // Turn on the verbose level logging by default.
-    private LogLevel mLogLevel = LogLevel.Verbose;
     private ILogger mExternalLogger = null;
-    private static final String CUSTOM_LOG_ERROR = "Custom log failed to log message:%s";
-    private boolean mAndroidLogEnabled = false;
-    private String mCorrelationId = null;
 
-    // Disable to log PII by default.
-    private boolean mEnablePII = false;
+    private String mCorrelationId = null;
 
     /**
      * @return The single instance of {@link Logger}.
+     *
+     * @deprecated use {@link com.microsoft.identity.common.internal.logging.Logger#getInstance()} instead.
      */
+    @Deprecated
```

Comment:
why is this deprecated? How will the apps get the instance of the logger? apps should not call into common core